pom.xml

```diff
@@ -121,9 +121,6 @@
         -->
         <profile>
             <id>it-basic</id>
-            <activation>
```

Comment:
Was this property redundant or no are you trying to make it un-active by default ?

Reply:
This was temporarily introduced in 5db7d3b to fix CI

Reply:
Okay. Gotcha!
This profile was meant to test only the basic-ITs in the default build.
Thanks for explaining.